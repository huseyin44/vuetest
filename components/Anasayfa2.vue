<template>
   <div>
      <img class="resim1" height="300" width="1550"  src="https://cdn.saatvesaat.com.tr/media/wysiwyg/Bulk/online-ozel-07-10.jpg" >
  </div>
</template>

<script>
export default {

}
</script>

<style>

</style>