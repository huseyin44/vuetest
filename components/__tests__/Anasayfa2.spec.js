import { shallowMount } from "@vue/test-utils";
import Anasayfa2 from "../Anasayfa2";
const factory = () => {
    return shallowMount(Anasayfa2, {
    });
  };
  
  describe("Anasayfa2", () => {
    test("Bir Vue kopmonenti", () => {
      const wrapper = factory();
      expect(wrapper.isVueInstance()).toBeTruthy();
    });

    test("Snapshot Ozelligi ", () => {
      const wrapper = factory();
      expect(wrapper.html()).toMatchSnapshot();
    });
  
  });