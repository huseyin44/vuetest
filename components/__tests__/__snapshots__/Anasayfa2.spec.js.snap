// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Anasayfa2 Snapshot Ozelligi  1`] = `<div><img height="300" width="1550" src="https://cdn.saatvesaat.com.tr/media/wysiwyg/Bulk/online-ozel-07-10.jpg" class="resim1"></div>`;
